app/src/main/java/org/mozilla/focus/FocusApplication.java

```diff
@@ -6,9 +6,12 @@
 package org.mozilla.focus;
 
 import android.arch.lifecycle.LiveData;
+import android.content.Context;
```

Comment:
\[Checkstyle\] ERROR: Unused import \- android\.content\.Context\.

Reply:
Thanks, resolved it :)